src/components/Button/Button.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styles from './Button.scss';
+import { Button as StyledButton } from './Button.Styles';
```

Comment:
you mixed it up :)

`import * as Styled from './styles'` or `from './Button.Styles'` - doesn't matter, but I think the first variant is better
then 
`<Styled.Button>...`